src/Microsoft.AspNet.Server.Kestrel/Networking/Libuv.cs

```diff
@@ -85,7 +87,7 @@ public int Check(int statusCode, out Exception error)
 
         [UnmanagedFunctionPointer(CallingConvention.Cdecl)]
         delegate int uv_loop_init(UvLoopHandle a0);
-        uv_loop_init _uv_loop_init;
+        uv_loop_init _uv_loop_init = default(uv_loop_init);
```

Comment:
either works :)

kind of nice to see warnings if other code in the file references an uninitialized thing